### PR TITLE
Fixup sync extension worker crash

### PR DESCRIPTION
**Pull request by jamesdphillips**
Handles nil case when an extension does not contain any YAML or Ruby
files.

### CODE REVIEW

File: app/workers/sync_extension_contents_at_versions_worker.rb [hunk 1]
```diff
@@ -203,8 +203,8 @@ def extract_readme_file_extension(filename)
   end
 
   def ensure_updated_version(readme_body, readme_ext)
-    yml_line_count = @run.cmd("find . -name '*.yml' -o -name '*.yaml' -print0 | xargs -0 wc -l").split("\n").last || ""
-    rb_line_count = @run.cmd("find . -name '*.rb' -print0 | xargs -0 wc -l").split("\n").last || ""
+    yml_line_count = @run.cmd("find . -name '*.yml' -o -name '*.yaml' -print0 | xargs -0 wc -l")&.split("\n")&.last || ""
+    rb_line_count = @run.cmd("find . -name '*.rb' -print0 | xargs -0 wc -l")&.split("\n")&.last || ""
```
Comment (jamesdphillips) [inline app/workers/sync_extension_contents_at_versions_worker.rb]: ... because my editor went and fixed up unnecessary whitespace, the fix can be found here.

Comment (portertech) [reply app/workers/sync_extension_contents_at_versions_worker.rb]: TIL http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/

**Comment by portertech** [comment]
sandcastle

File: app/workers/sync_extension_contents_at_versions_worker.rb (hunk 1 above)
Comment (calebhailey) [reply app/workers/sync_extension_contents_at_versions_worker.rb]: WUT?! 

That would have saved me so much time! 🤦🏾‍♂️
